packages/ui/vanilla-ui/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@crossmint/client-sdk-vanilla-ui",
+  "version": "0.0.1",
+  "description": "Vanilla Crossmint button using web-components",
```

Comment:
Crossmint client SDK for vanilla JS (no framework required), using web-components